src/Link/index.js

```diff
@@ -32,6 +32,15 @@ export default class Link extends React.PureComponent {
     }
   }
 
+  drawStairPath(linkData, orientation) {
+    const { source, target } = linkData;
+    const deltaY = target.y - source.y;
+
+    return orientation === 'horizontal'
+      ? `M${source.x},${source.y} V${source.y + deltaY / 2} H${target.x} V${target.y}`
```

Comment:
While test-driving your change on the demo I noticed that this ternary is the wrong way around, i.e. the nodes were rendering horizontally but the links vertically.

Flipping the ternary expression results in correct behaviour though so an easy fix :)

Reply:
Yeah, thanks. I'll fix it.

I have question how to run demo for fast visual changes. Can you describe it?

Reply:
Yeah apologies if the demo is confusing; it's really not properly set up for public usage at the moment.

I've just updated the readme on the demo to simplify the process somewhat with `yarn/npm link`: https://github.com/bkrem/react-d3-tree-demo/

Basically, I symlink the library into the demo and have both running in watch mode (webpack on the lib, standard `nodemon` for the demo) to hot reload changes automatically.

Let me know if something is still unclear there or incorrect. Cleaning up the demo and making it more friendly for people to experiment with is one of the multiple issues I'm planning to address for this library when I'm on sabbatical early 2020.